### PR TITLE
Activity list

### PR DESCRIPTION
**Pull request by sinafischer**

### CODE REVIEW

File: src/components/ActivityListItem.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import styled from 'styled-components'
+import ShowMoreText from 'react-show-more-text'
+
+export default function ActivityList() {
+  return (
+    <>
+      {activities.map(activity => {
+        return (
+          <li key={activity.name}>
+            <DivisionLine data-testid="division-line" />
```
Comment (miriamhaenle) [inline src/components/ActivityListItem.js:12]: Damit du in der Attribut-Schreibweise konsistent bleibst, würde ich data-test-id schreiben

File: src/components/ActivityListItem.test.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import '@testing-library/jest-dom/extend-expect'
+import React from 'react'
+import ActivityListItem from './ActivityListItem'
+import { render, screen } from '@testing-library/react'
+
+describe('ActivityListItem.test.js', () => {
```
Comment (miriamhaenle) [inline src/components/ActivityListItem.test.js:6]: Bei den Tests hier verstehe ich die Assertion nicht ganz. Muss man immer mindestens 4 Stück hinzufügen? Was wäre wenn es noch keine gibt? Vielleicht findest du noch ein paar Edge Cases :)

**Comment by sinafischer** [comment]
Hi Miriam,
vielen Dank für die super Code-Review! Freue mich über jede Anmerkung. :-)

File: src/components/Header.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import styled from 'styled-components'
+import globe from '../images/globe.svg'
+
+export default function Header() {
+  return (
+    <StyledHeader>
+      <img src={globe} alt="" />
+      <span>VacationDetails</span>
```
[the remark this answers is not in the document]
Comment (sinafischer) [reply src/components/Header.js:9]: Ja, hatte es aus Styling-Gründen gemacht. Aber da gibt es sicherlich noch ne andere Lösung. Falls ich keine finde, werde ich dann auf das von dir vorgeschlagene aria-label setzen.

File: src/components/Header.test.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+    render(<Header />)
+    expect(screen.getByText('VacationDetails')).toBeInTheDocument()
+  })
+
+  it('should exist an image', () => {
+    render(<Header />)
+    expect(screen.getByRole('img')).toBeInTheDocument()
+  })
+
+  it('should show an image', () => {
```
[the remark this answers is not in the document]
Comment (sinafischer) [reply src/components/Header.test.js:17]: Ja, stimmt. Werde ich anpassen. :-)

File: src/components/ActivityListItem.test.js (hunk 2 above)
Comment (sinafischer) [reply src/components/ActivityListItem.test.js:6]: Ich wollte testen, ob für alle fünf activitites aus der json-Datei ein mapping und rendering stattfindet, daher die toBeGreaterThan(4). Im Grunde hätte es vermutlich gereicht einfach auf Existenz zu testen und dann toBeGreaterThan(0).

File: src/components/ActivityListItem.js (hunk 1 above)
Comment (sinafischer) [reply src/components/ActivityListItem.js:12]: Gerade angepasst und dann einen Fehler bekommen, dass nichts über data-test-id gefunden wird. Anscheinend ist data-testid die korrekte/notwendige Schreibweise.
